plugin-server/src/worker/vm/lazy-vm/lazy.ts

```diff
@@ -173,29 +162,23 @@ export class LazyPluginVM {
     }
 
     public async setupPluginIfNeeded(): Promise<boolean> {
```

Comment:
I noticed a few methods such as, `clearRetryTimeoutIfExists` and `setupPluginIfNeeded`.
A powerful paradigm is rephrasing these types of methods as, `ensureTimeoutCleared` or `ensurePluginSetup`.

>I made this mistake myself in the design of the Tcl scripting language. Tcl contains an unset command that can be used to remove a variable. I defined unset so that it throws an error if the variable doesn’t exist. At the time I thought that it must be a bug if someone tries to delete a variable that doesn’t exist, so Tcl should report it. However, one of the most common uses of unset is to clean up temporary state created by some previous operation. It’s often hard to predict exactly what state was created, particularly if the operation aborted partway through. Thus, the simplest thing is to delete all of the variables that might possibly have been created. The definition of unset makes this awkward: developers end up enclosing calls to unset in catch statements to catch and ignore errors thrown by unset . In retrospect, the definition of the unset command is one of the biggest mistakes I made in the design of Tcl.
>A Philosophy of Software Design